build/images/ovs/Dockerfile

```diff
@@ -21,7 +21,7 @@ ARG OVS_VERSION
 # Install dependencies for building OVS deb packages
 # We only install python3 packages and we only support building OVS >= 2.13.0.
 RUN apt-get update && \
-    DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends wget curl git ca-certificates build-essential fakeroot graphviz \
+    DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends wget curl git ca-certificates build-essential fakeroot \
```

Comment:
Not sure if this is actually required by other packages instead of Octant. @antoninbas is this OK to be removed?

Reply:
I think it's ok to remove it. I think we haven't needed this package for a long time... even before removing the Octant plugin.